src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -2389,7 +2389,7 @@ int RGWRados::get_obj_head_ioctx(const DoutPrefixProvider *dpp,
     return -EIO;
   }
 
-  int r = open_pool_ctx(dpp, pool, *ioctx, false);
+  int r = open_pool_ctx(dpp, pool, *ioctx, false, false);
```

Comment:
`get_obj_head_ioctx()` refers to a data pool, so this should pass bulk=true

Reply:
ack